<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cBudget</class>
 <widget class="QMainWindow" name="cBudget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
     </font>
    </property>
    <property name="text">
     <string>Budget Category:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
     </font>
    </property>
    <property name="text">
     <string>Set Limit:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
     </font>
    </property>
    <property name="text">
     <string>Set reminder when total spent hits:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
     </font>
    </property>
    <property name="text">
     <string>Create Budget</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>310</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
     </font>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCreate_Budget">
    <property name="title">
     <string>Create Budget</string>
    </property>
   </widget>
   <addaction name="menuCreate_Budget"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
